opencog/cython/opencog/type_constructors.pyx

```diff
@@ -0,0 +1,132 @@
+
+# DO NOT EDIT THIS FILE! This file was automatically generated from atom
+# definitions in types.script by the macro OPENCOG_ADD_ATOM_TYPES
```

Comment:
If I understand correctly, this .pyx file is dynamically generated by the cmake file when the project is built? And if so, why does it need to be committed? Or maybe I misunderstood?